### PR TITLE
Synchronize with Transaction Input Stoa

### PR DESCRIPTION
**Pull request by TrustHenry**
previous to `utxo` & remove the `index`

Related to #49

### CODE REVIEW

File: src/modules/data/TxInput.ts [hunk 1]
```diff
@@ -22,14 +22,9 @@ import { SmartBuffer } from 'smart-buffer';
 export class TxInput
 {
     /**
-     * The hash of the previous transaction containing the output to spend
+     * The hash of the utxo transaction containing the output to spend
```
Comment (MichaelKim20) [inline src/modules/data/TxInput.ts:25]:
```suggestion
     * The hash of the UTXO to be spent
```

File: src/modules/data/TxInput.ts [hunk 2]
```diff
@@ -38,21 +33,15 @@ export class TxInput
 
     /**
      * Constructor
-     * @param previous The hash of the previous transaction containing the output to spend
-     * @param index The index of the output in the previous transaction
+     * @param utxo The hash of the utxo transaction containing the output to spend
```
Comment (MichaelKim20) [inline src/modules/data/TxInput.ts:36]:
```suggestion
     * @param utxo The hash of the UTXO to be spent
```

**Review by MichaelKim20 — COMMENTED** [review]
Everything's fine.
I'd like to change the comment I suggested.

**Comment by TrustHenry** [comment]
@MichaelKim20 I apply the suggestions

**Review by MichaelKim20 — APPROVED** [review]

**Pull request merged**